packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -113,10 +113,6 @@ const LOCATION_VIEW_COLUMNS: Column<LocationAccess<Permission>>[] = [
     header: 'Name',
     key: 'scope',
   },
-  {
-    header: 'Type',
-    key: 'type',
-  },
```

Comment:
Why are we removing this

Reply:
I think it was originally an ask from Scott since the locations view type column only shows: `PREFIX`

Would it ever show anything else?

![image](https://github.com/user-attachments/assets/5fbf4cd7-a943-44ce-b280-0bcad7b4a705)

Reply:
Have you been testing the managed auth app? The default auth app only has prefixes, but that is not the case with access grants

Reply:
ah I have been testing it, but didn't notice that it was different on the managed auth app. I can leave it in and close this PR then.